packages/core/src/render3/instructions/defer.ts

```diff
@@ -100,7 +104,7 @@ export function ɵɵdefer(
 
   // Init instance-specific defer details and store it.
   const lDetails = [];
-  lDetails[DEFER_BLOCK_STATE] = DeferBlockInstanceState.INITIAL;
+  lDetails[DEFER_BLOCK_STATE] = DeferBlockInitialState.Initial;
```

Comment:
nit: I think we can rename `DeferBlockInitialState` -> `DeferBlockInternalState`, which would also reflect the logic better:
```suggestion
  lDetails[DEFER_BLOCK_STATE] = DeferBlockInternalState.Initial;
```